src/tests/ftest/osa/osa_online_reintegration.yaml

```diff
@@ -0,0 +1,48 @@
+server_manager:
```

Comment:
This file has the mix of 2-space and 4-space indentation. It would be great if you could make it consistent.

Reply:
Done.